.github/workflows/e2e_test_ci.yml

```diff
@@ -0,0 +1,249 @@
+name: E2E Test CI
+
+on:
+  push:
+    branches:
+      - trunk
+      - release-*
+
+  pull_request:
+    branches:
+      - trunk
```

Comment:
TODO: disable trigger on PR, this should be run only on push to trunk/release branches, or manually

Reply:
removed